pkg/npm/api/hood/lib.ts

```diff
@@ -87,35 +84,6 @@ export function kilnResume(desk: string) {
 
 export const scryLag: Scry = ({ app: 'hood', path: '/kiln/lag' });
 
-export function getBlockers(vats: Vats): string[] {
```

Comment:
Any reason why we're not making the equivalent here?

Reply:
Removed this because as far as I can tell it's not being consumed anywhere downstream. Can restore it if there's a possibility that it might be used again in grid or elsewhere.